samples/routing/data-distribution/Core_6/Server/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+class Program
+{
+
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.DataDistribution.Server";
+        LogManager.Use<DefaultFactory>()
+            .Level(LogLevel.Info);
```

Comment:
Sorry, copied from the fullduplex sample again and forgot to remove. I'll remove it there also (eventually).